src/shared/components/ui/LimitSmall/LimitSmall.js

```diff
@@ -1,23 +1,27 @@
 import { formatNumeric } from '@helpers/data'
-import { styles } from '@helpers/css'
 import PropTypes from 'prop-types'
 import React from 'react'
-import scss from './LimitSmall.scss'
-
-const cssClass = styles(scss)
+import styles from './LimitSmall.modules.scss'
+import { useStyles } from '@helpers/hooks/useStyles'
 
 /**
  * Limit Small - stateless presentational component
  * @param {object} props - props
+ * @param {string|array} props.className - list of class names
  * @param {string} props.padding - top padding size, default 'none'
  * @param {number} props.limit - limit to display
  * @param {string} props.quantity - quantity to display
  * @param {string} props.limitText - limit type info
  * @return {object} An object of children element
  */
-const LimitSmall = ({ padding, limit, quantity, limitText }) => {
+const LimitSmall = ({ className, padding, limit, quantity, limitText }) => {
+  const limitSmallStyles = useStyles({
+    [className]: true,
```

Comment:
To się wysypie, gdy ktoś poda swoje customowe klasy w propsie className więcej niż jedną i nie uwzględni w nich takiej jak napisałeś w `defaultProps: styles.result__dropdown`

Rozwiązanie tu gdzie masz className przekaż jawnie [styles[`result__dropdown`]: true, a cały props className daj jako drugi argument do useStyles, tak aby każdy mógł przekazywać w przyszłości osobne customowe klasy od modułowych :D